server.py

```diff
@@ -1,92 +1,108 @@
 #!/usr/bin/env python3
 
-from bottle import (
-    get, post, run, static_file, template, request, response, redirect
-)
-
-from base64 import urlsafe_b64decode as b64dec
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization, hashes
+from cryptography.hazmat.primitives.asymmetric import rsa, padding
+from base64 import urlsafe_b64decode
 import json
 from os import getenv
-from random import getrandbits
 import re
+from urllib import parse, request
 from time import time
+from wsgiref import simple_server
+import os, mimetypes
 
-SECRET = getenv('COOKIE_SECRET', '%x' % getrandbits(128))
+DIR = os.path.dirname(os.path.abspath(__file__))
 
 META = {
-    'LA_HOSTNAME': 'laoidc.herokuapp.com',
-    'RP_ORIGIN': 'http://localhost:%s' % getenv('PORT', '8080'),
+    'LA_ORIGIN': 'https://letsauth.xavamedia.nl',
+    'RP_ORIGIN': 'http://xavamedia.nl:%s' % getenv('PORT', '8000'),
 }
```

Comment:
We should probably not hardcode your domain here. Or we should use a proper configuration mechanism. :)